src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/GeneralViewModel.cs

```diff
@@ -36,7 +36,7 @@ public class GeneralViewModel : Observable
         public GeneralViewModel(ISettingsRepository<GeneralSettings> settingsRepository, string runAsAdminText, string runAsUserText, bool isElevated, bool isAdmin, Func<string, int> updateTheme, Func<string, int> ipcMSGCallBackFunc, Func<string, int> ipcMSGRestartAsAdminMSGCallBackFunc, Func<string, int> ipcMSGCheckForUpdatesCallBackFunc, string configFileSubfolder = "")
         {
             CheckFoUpdatesEventHandler = new ButtonClickCommand(CheckForUpdates_Click);
-            RestartElevatedButtonEventHandler = new ButtonClickCommand(Restart_Elevated);
+            RestartElevatedButtonEventHandler = new ButtonClickCommand(RestartElevated);
```

Comment:
Why is this one RestartElevated and the other is CheckForUpdates_Click

Reply:
This warning is only applied to public methods and I just noticed the other one is private for some reason. I'll just remove the underscore from that one